lib/auth-logout.ts

```
@@ -58,7 +58,7 @@ export async function serverLogoutAndCleanup(): Promise<void> {
   try {
     // Server-side cookie clearing should be handled by auth API routes directly
     // This function is kept for compatibility but doesn't do server-side operations
-  } catch (error) {
-    throw error;
+  } catch (_error) {
+    throw _error;
   }
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)